WalletWasabi/Blockchain/Analysis/FeesEstimation/AllFeeEstimate.cs

```diff
@@ -52,6 +53,48 @@ public AllFeeEstimate(AllFeeEstimate other, bool isAccurate)
 		[JsonProperty]
 		public Dictionary<int, int> Estimations { get; }
 
+		/// <returns>
+		/// An estimates based on these estimates, but randomizes each estimation between
+		/// the value of its larger estimation sibling - or 50% more if the largest
+		/// with a probabilistic bias towards the estimate.
+		/// </returns>
+		public AllFeeEstimate? Unfingerprint()
+		{
+			if (Estimations is null)
```

Comment:
It does not seem that `Estimations` can be `null`.

Reply:
If it is possible then do not return `null`.